sites/friday/src/get-started/quick-start/use-third-part.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Use a third party provider
```

Comment:
We should probably title, and refer to this section as "Source integrations" since that is how we refer to it [everywhere else in the docs](https://docs.upsun.com.fix-generic-gstarted-7jc6g4y-652soceglkw4u.eu-3.platformsh.site/integrations/source.html).